Huion_WH1409/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: WH1409
+image: Huion_WH1409
```

Comment:
Can you find a better quality image? It's hard to see what the buttons are on this one.